web/app/js/components/Tap.jsx

```diff
@@ -162,73 +158,12 @@ class Tap extends React.Component {
     };
   }
 
-  getFilterOptions(d) {
```

Comment:
I think we also may need to remove `getInitialTapFilterOptions`. I don't see it being used anywhere else.